cranelift/interpreter/src/value.rs

```diff
@@ -11,9 +11,8 @@ use thiserror::Error;
 
 pub type ValueResult<T> = Result<T, ValueError>;
 
-pub trait Value: Clone + From<DataValue> {
+pub trait DataValueExt: Sized {
```

Comment:
Left this as extension trait to make this PR smaller. We can always move methods to `DataValue` directly or inline them at the call site in cranelift-interpreter in the future.